src/bundle/Resources/public/scss/_breadcrumb.scss

```diff
@@ -4,6 +4,9 @@
 
 .breadcrumb-item {
     cursor: not-allowed;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
When we add ellipsis we also should add `title` attribute to make sure that people can read the whole text.
This one applies to all places where you added the ellipsis